draft-ietf-quic-recovery.md

```diff
@@ -619,18 +623,24 @@ are as follows:
   ACK frame.  If true, it is still expected an ack will be received for
   this packet, but it is not congestion controlled.
 
+* is_handshake_packet: A boolean that indicates whether a packet is part of
```

Comment:
How does this apply to ack-only packets sent during the handshake?

It we're regarding them as handshake packets here, we probably need to change `OnPacketSend`, such that `time_of_last_sent_handshake_packet` is only updated for retransmittable handshake packets.

Reply:
Ah, I was thinking a handshake packet was a packet with handshake data in it, not based on packet type, but I'll restructure to make it clear.